src/core/server/saved_objects/migrations/actions/calculate_exclude_filters.ts

```diff
@@ -91,22 +91,17 @@ export const calculateExcludeFilters =
       }
 
       const errorsByType: Array<[string, Error]> = [];
-      const filters: estypes.QueryDslQueryContainer[] = [];
+      const mustNotClauses: QueryDslQueryContainer[] = [];
 
       // Loop through all results and collect successes and errors
       results.forEach((r) =>
         Either.isRight(r)
-          ? filters.push(r.right)
+          ? mustNotClauses.push(r.right)
           : Either.isLeft(r) && errorsByType.push([r.left.soType, r.left.error as Error])
       );
 
-      // Composite filter from all calculated filters that successfully executed
-      const excludeFilter: estypes.QueryDslQueryContainer = {
-        bool: { must_not: filters },
-      };
-
       return Either.right({
-        excludeFilter,
+        mustNotClauses,
```

Comment:
We no longer encapsulate the received filters inside a `bool.must_not`. That becomes a responsibility of the caller now.